src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -43,17 +43,22 @@ StatusNotifierItem::StatusNotifierItem(QString id, QObject *parent)
     mTitle("Test"),
     mStatus("Active"),
     mMenu(nullptr),
-    mMenuExporter(nullptr)
+    mMenuExporter(nullptr),
+    mSessionBus(QDBusConnection::connectToBus(QDBusConnection::SessionBus, mService))
 {
+    // we created a separate DBus connection, because multiple dbus services with _different_
+    // behavior are not supported on QDbus
```

Comment:
I dug a bit into it... and now I think the comment here is not 100% valid.

The problem here is/was, that the `QDBusConnection` does not support multiple objects with the same name (here "/statusNotifierItem") -> the `QDBusConnection::registerObject()` does not provide such API. 

So in the old implementation all services (`org.freedesktop.StatusNotifierItem-{pid}-{cnt}`) were registered on one `QDBusConnection`, but only the first `StatusNotifierItem` object was registered successfully. => all method calls from lxqt-panel(`org.kde.StatusNotifierHost-{pid}-1`) were delivered to object registered as first.